portal/src/locale-data/en.json

```diff
@@ -276,6 +276,8 @@
   "PasswordlessAuthenticatorScreen.subsequent-logins.sms-body.label": "SMS body for subsequent logins",
 
   "OAuthClientConfiguration.title": "Client Applications",
+  "OAuthClientConfigurationScreen.api-endpoint.header": "API Endpoint",
```

Comment:
Remove the term API from everything done in this PR. The term API is a bit confusing.